flink-runtime/src/test/java/org/apache/flink/runtime/util/profiler/ProfilingServiceTest.java

```diff
@@ -125,10 +128,18 @@ public void testRollingDeletion() throws ExecutionException, InterruptedExceptio
      * @param waitUntilFinished flag of waiting for profiling finished or not
      */
     private void requestSingleProfiling(
-            ProfilingInfo.ProfilingMode mode, Long duration, Boolean waitUntilFinished)
+            TestInfo testInfo,
+            ProfilingInfo.ProfilingMode mode,
+            Long duration,
+            Boolean waitUntilFinished)
             throws InterruptedException, ExecutionException {
         ProfilingInfo profilingInfo =
-                profilingService.requestProfiling(RESOURCE_ID, duration, mode).get();
+                profilingService
+                        .requestProfiling(
+                                RESOURCE_ID + "_" + testInfo.getTestMethod().get().getName(),
```

Comment:
just use test name as part of prefix to make it unique per test and to avoid tests impacting each other while concurrent execution